assets/src/edit-story/components/panels/preset/presetPanel.js

```diff
@@ -136,6 +136,7 @@ function PresetPanel({
       initialHeight={Math.min(initialHeight, window.innerHeight / 3)}
       resizeable={resizeable}
       canCollapse={canCollapse}
+      persistenceKey={title}
```

Comment:
That's error prone because `title` is translatable. I'd suggest just using `name` here.

Reply:
Yea, I figured this was such a small edge & would happen so infrequently it'd be fine for now. Happy to key off a different value tho if you see one that better suits it

Reply:
https://github.com/google/web-stories-wp/pull/4601#discussion_r492287834

Reply:
It means the panel states will change whenever users change their preferred UI language or whenever the translation changes. The latter will probably be a bit more frequent especially after our initial release, as users will start to translate the plugin.